Spore ModAPI/Spore/Simulator/cScenarioPlayMode.h

```diff
@@ -67,9 +67,14 @@ namespace Simulator
 		/* 64h */	eastl::vector<cScenarioPlayModeGoal> mCurrentGoals;
 		/* 78h */	int field_78;  // not initialized
 		/* 7Ch */	App::MessageListenerData mMessageListenerData;
-		/* 90h */	int field_90;  // not initialized, current state? fail reason?
-		/* 94h */	int field_94;  // not initialized
-		/// The clock activates upon triggering the adventure's end. It counts up to around 2000 units (milliseconds),  after which the ending cinematic activates
+		/// Current gameplay state of the adventure.
+		/// 0 is pre-init in editor play mode, 1 is intro, 2 seems to be pre-init in quick-play, 3 is gameplay, 5 is adventure completed and 6 is adventure failed (death or failed/invalid goals)
+		/// Not initialized
+		/* 90h */	int mCurrentGameplayState;
```

Comment:
You can create an `enum` with those possible values. If you know these are all the possible values, you can use `enum class` and declare `mCurrentGameplayState` with that type; if there can be more possible values, just leave it as an `int` and mention the enum type in the documentation comment.

Also, leave the `// not initialized` comment like before. These are just internal comments, no need to be part of the documentation.